### ISSUE TITLE
Specify Python version compatibility

### ISSUE THREAD

**Issue by BVasquez07** [whylabs/langkit#261]
Had an extremely difficult time trying install the Langkit module with a venv built with python version 3.12.1 on Windows 10 64-bit machine. There was an error with building the whylogs-sketching wheel on install. Another grievance is that in the mac terminal there should be clear alternates to how the pip command is written with it being pip install "langkit[all]"

**Comment by jamie256** [comment]
Agreed, sorry for the trouble. We will publish a python3.12 version of whylogs-sketching, and good to restrict the python version to less than 3.12.

**Comment by ddorn** [comment]
I had the same trouble, and would love to see that happen. I'm happy to open a PR to update the version constraint if desired

**Comment by vargoson** [comment]
Well, glad to see the issue was resolved after a year and half. (It was not.)
